src/Core/Framework/Adapter/Twig/TokenParser/MacroOverrideTokenParserMacro.php

```diff
@@ -5,15 +5,13 @@
 use Shopware\Core\Framework\Log\Package;
 use Twig\Token;
 
-#[Package('framework')]
 /**
- * @internal
```

Comment:
why do we remove the internal here?

Reply:
I guess BC Checker was unhappy about that 🤔 But I can add and exclude it again.

Reply:
we should make sure, that every class in the twig adapter namespace is internal. in general all classes in the adapter namespace should be internal